test/parallel/test-child-process-fork-regr-gh-2847.js

```diff
@@ -18,6 +18,11 @@ if (!cluster.isMaster) {
 }
 
 var server = net.createServer(function(s) {
+  s.on('error', function(err) {
```

Comment:
Maybe wrap this in an `if (common.isWindows)` so it only applies to the affected OS? Or not. Just throwing it out there.